Package.swift

```diff
@@ -27,7 +27,7 @@ let package = Package(
         //
         // Note: using `.exact("")` instead of `from: ""` because 3.0.3 will not compile.
         // Using 3.0.2 for now to not allow upgrading.
```

Comment:
Should probably update the comment as well to reflect the current state of things, right?

Reply:
updated